src/viperleed/calc/sections/run_sections.py

```diff
@@ -39,6 +40,7 @@
 
 
 logger = logging.getLogger(__name__)
+R_MAX = np.inf
```

Comment:
```suggestion
R_MAX = float('inf')
```

This way no need for numpy